releases/sui/2025-09-ITS-v1.2.0.md

```diff
@@ -166,10 +158,6 @@ Test Sui Token Deployment using either lock / unlock or mint / burn token manage
 # Register token with manager type: lock / unlock
 ts-node sui/its-example deploy-token --origin TST "Test Token" 6
 ```
-```bash
-# Register token with manager type: mint / burn
```

Comment:
Fix the script as we had discussed by pre minting tokens instead of removing testing for mint/burn type

Reply:
See #1034